### DISCUSSION TITLE
Unable to Signup or login at vercel

### DISCUSSION THREAD

**Discussion by Safoora-Iftikhar** [vercel/vercel#10450] [Help]
### Question

[](url)I created a new Vercel account using the my gmail address. Additionally, I attempted to sign up using my GitHub account associated with this email. However, I'm unable to log in or sign up successfully.
When I use this email for sign-up, it displays an error message:

![error1](https://github.com/vercel/vercel/assets/122617267/dd0d42d9-7e85-4608-87a4-a170b3d11e8a)

when I attempt to log in, I receive an error message:
![error2](https://github.com/vercel/vercel/assets/122617267/8152df1d-3933-48f4-b512-c47a22e3f077)

I would greatly appreciate any assistance in resolving this issue. Thank you in advance for your help!

**Comment by alvinsjoy** [comment]
Vercel might have detected you as a bot or scam account you will have to contact [support@vercel.com](mailto:support@vercel.com) and they'll unblock your account.
Re: #10000
![image](https://github.com/vercel/vercel/assets/89687635/8b00db36-c881-433a-b8ec-1c4b49e54c9d)
